forms/NumericField.php

```diff
@@ -30,7 +30,7 @@ public function setValue($value, $data = array()) {
 		// If passing in a non-string number, or a value
 		// directly from a DataObject then localise this number
 
-		if(is_int($value) || is_float($value) || $data instanceof DataObject) {
+		if(is_numeric($value) || is_int($value) || is_float($value) || $data instanceof DataObject) {
```

Comment:
I do agree, as mentioned above, changing this causes issues with localisation. I will try to suggest a different PR.